server/protocols/jmap-rfc-8621/pom.xml

```diff
@@ -56,9 +54,9 @@
             <version>0.9.13</version>
         </dependency>
         <dependency>
-            <groupId>de.cbley</groupId>
-            <artifactId>play-json-refined_${scala.base}</artifactId>
-            <version>0.8.0</version>
+            <groupId>be.venneborg</groupId>
+            <artifactId>play27-refined_2.13</artifactId>
+            <version>0.5.0</version>
```

Comment:
It's this right? https://github.com/kwark/play-refined

Why not, sounds like simplifying the code right? :)

Reply:
Yes this is what we want.